### ISSUE TITLE
LuaJit

### ISSUE THREAD

**Issue by Bagerter** [theraw/The-World-Is-Yours#26]
Added in Nginx.conf 

nginx: the configuration file /nginx/nginx.conf syntax is ok
nginx: configuration file /nginx/nginx.conf test is successful

but when im restart I get this error

ginx: [warn] lua_load_resty_core is deprecated (the lua-resty-core lib>
Dec 21 07:36:48 ubuntu-32gb-hel1-2 nginx[195503]: PANIC: unprotected error in call to Lua API (attempt to index a boolean>
Dec 21 07:36:48 ubuntu-32gb-hel1-2 systemd[1]: nginx.service: Control process exited, code=dumped, status=6/ABRT
Dec 21 07:36:48 ubuntu-32gb-hel1-2 systemd[1]: nginx.service: Failed with result 'core-dump'.
Dec 21 07:36:48 ubuntu-32gb-hel1-2 systemd[1]: Failed to start nginx.service.

lua_load_resty_core is deprecated (the lua-resty-core library is required since ngx_lua v0.10.16) in /nginx/nginx.conf>
otected error in call to Lua API (attempt to index a boolean value)
 Control process exited, code=dumped, status=6/ABRT
 Failed with result 'core-dump'.
t nginx.service.

**Comment by theraw** [comment]
can you post `lsb_release -a; uname -a` ?

**Comment by Bagerter** [comment]
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.5 LTS
Release:        20.04
Codename:       focal
Linux ubuntu-32gb-hel1-2 5.4.0-135-generic #152-Ubuntu SMP Wed Nov 23 20:19:22 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux



Added in nginx.conf

load_module /nginx/modules/ndk_http_module.so; 
load_module /nginx/modules/ngx_http_lua_module.so;
http {
    lua_load_resty_core off;
    lua_package_path "/usr/local/LuaJIT/?.lua;;";

**Comment by theraw** [comment]
delete lua_load_resty_core off;
every module installed here has its own documentations in their repos example https://github.com/openresty/lua-nginx-module or https://www.nginx.com/resources/wiki/modules/lua/
read there if you don't know what you're doing..

**Comment by Bagerter** [comment]
Dec 24 18:12:54 ubuntu-32gb-hel1-2 nginx[52799]: nginx: the configuration file /nginx/nginx.conf syntax is ok
Dec 24 18:12:54 ubuntu-32gb-hel1-2 nginx[52799]: nginx: configuration file /nginx/nginx.conf test is successful
Dec 24 18:12:54 ubuntu-32gb-hel1-2 nginx[52807]: PANIC: unprotected error in call to Lua API (attempt to index a boolean value)
Dec 24 18:12:54 ubuntu-32gb-hel1-2 systemd[1]: nginx.service: Control process exited, code=dumped, status=6/ABRT

**Comment by theraw** [comment]
yeah well lua-nginx developers made openresty and now they want to forcefully push ppl to install openresty.

i already fixed this once idk what's up now but i'll take a look when i get free time

**Comment by Bagerter** [comment]
good im waiting ty

**Comment by theraw** [comment]
just released a patch for ubuntu 20.04 give it a try
curl -s https://raw.githubusercontent.com/theraw/The-World-Is-Yours/master/install > install; bash install

make sure its a fresh server, you can also try to execute that script in docker container as long as its ubuntu 20.04.
after you add lua module .so make sure you uncomment this https://github.com/theraw/The-World-Is-Yours/blob/master/static/Focal/nginx.conf#L21
from
```
#lua_package_path "/usr/twiylua/lib/lua/?.lua;;";
```
to
```
lua_package_path "/usr/twiylua/lib/lua/?.lua;;";
```

i will fix ubuntu 22.04 tomorrow if i have time.
